### PR TITLE
Update CI to build from and generate 3.0.1.

### PR DESCRIPTION
**Pull request by licquia**

### CODE REVIEW

**Comment by goneall** [comment]
Note the more recent PR #950 - we should probably merge these two PRs into a single PR

File: .github/workflows/publish_v3.yml [hunk 1]
```diff
@@ -39,15 +39,15 @@ jobs:
...
       - name: Set git identity
         run: git config user.name ci-bot; git config user.email ci-bot@spdx.dev
         working-directory: spdx-spec
       - name: Sync gh-pages
-        run: git checkout gh-pages && git pull && git checkout development/v3.0
+        run: git checkout gh-pages && git pull && git checkout development/v3.0.1
         working-directory: spdx-spec
       - name: Build docs
-        run: mike deploy v3.0 v3-draft -b gh-pages -p
+        run: mike deploy v3.0.1 v3-draft -b gh-pages -p
```
Comment (bact) [inline .github/workflows/publish_v3.yml:52]: Change `v3-draft` to `latest`

**Comment by bact** [comment]
Probably can close this. PR #995 (merged) now publishes `development/v3.0.1` branch to  https://spdx.github.io/spdx-spec/v3.0.1-draft/

**Comment by kestewart** [comment]
Closing this as there is an alternate PR.

**Pull request closed without merging**